nf_core/pipeline-template/nextflow_schema.json

```diff
@@ -169,12 +176,6 @@
             "description": "Less common options for the pipeline, typically set in a config file.",
             "help_text": "These options are common to all nf-core pipelines and allow you to customise some of the core preferences for how the pipeline runs.\n\nTypically these options would be set in a Nextflow config file loaded for all pipeline runs, such as `~/.nextflow/config`.",
             "properties": {
-                "help": {
```

Comment:
Yep it's still a parameter but this will be automatically added by nf-schema now :) no need to add these manually. (they can still be added if it's necessary for other tools though)

Reply:
ok! 👏